### PR TITLE
BUG Fix 

### PR DESCRIPTION
**Pull request by prij**
issue where the current locale would not be assigned to the correct domain

### CODE REVIEW

**Comment by tractorcow** [comment]
Looks like a parse error in your PHP @prij . I guess I can fix it for you. :)

File: code/Fluent.php [hunk 1]
```diff
@@ -91,11 +91,14 @@ public static function current_locale($persist = true)
         // Check browser headers
         if (empty($locale)) {
             $locale = self::detect_browser_locale();
-        }
```
Comment (tractorcow) [inline code/Fluent.php]: I'm not sure this should have been dropped off?

**Comment by prij** [comment]
Woops sorry @tractorcow, I've just pushed that missing brace

File: code/Fluent.php [hunk 2]
```diff
@@ -92,10 +92,14 @@ public static function current_locale($persist = true)
         if (empty($locale)) {
             $locale = self::detect_browser_locale();
         }
-
-        // Fallback to default if empty or invalid
-        if (empty($locale) || !in_array($locale, self::locales())) {
-            $locale = self::default_locale();
+        
+        // Fallback to default if empty or invalid (for this domain)
```
Comment (tractorcow) [inline code/Fluent.php]: You also need to put this code back. :)

Comment (prij) [inline code/Fluent.php]: As we discussed :) Pushed.

**Comment by tractorcow** [comment]
Merged manually with https://github.com/tractorcow/silverstripe-fluent/commit/ceefc0a731585a1e296c85b0f0674e7b1e9e7d0d

**Pull request closed without merging**